docs/content/manual/_incoming/2229-2449-nic-ip-vip-different-harvester-installer copy.md

```diff
@@ -0,0 +1,22 @@
+---
+title: NIC ip and vip can't be the same in Harvester installer
+---
+
+* Related issues: [#2229](https://github.com/harvester/harvester/issues/2229) [BUG] input nic ip and vip with same ip address in Harvester-Installer
+* * Related issues: [#2449](https://github.com/harvester/harvester/issues/2449) [backport v1.0] [BUG] input nic ip and vip with same ip address in Harvester-Installer
+
+## Category: 
+* Harvester Installer
+
+## Verification Steps
+1. Launch ISO install process 
+1. Set static node IP and gateway
+![image](https://user-images.githubusercontent.com/29251855/173719118-1fd1609d-74f2-4f7d-9ff3-e1d21227e542.png)
```

Comment:
There should be a carriage return (newline) here and an indent to keep numbering consistent

Reply:
Thanks for the check, change line 13 with newline and a tab indent (4 characters wide)